pytorch_lightning/core/saving.py

```diff
@@ -350,7 +350,7 @@ def load_hparams_from_yaml(config_yaml: str, use_omegaconf: bool = True) -> Dict
         return {}
 
     with fs.open(config_yaml, "r") as fp:
-        hparams = yaml.load(fp, Loader=yaml.UnsafeLoader)
+        hparams = yaml.load(fp, Loader=yaml.Loader)
```

Comment:
@chris-boson Why replace this?

Reply:
Actually I think this does just default to UnsafeLoader, but it doesn't seem to cause problems with pyyaml 5.4.x functionality wise.
I think if we want to use SafeLoader we need to add our own Loader and add constructors https://github.com/yaml/pyyaml/issues/482#issuecomment-765737796

Reply:
Similar to this https://stackoverflow.com/a/52586430

Reply:
@Borda

Reply:
we are fine with declare own Loader, @chris-boson mind send a suggestion?